app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -2,14 +2,27 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class ConfigurationJWTGenericVendor < ConfigurationInterface
+      GENERIC_JWT_CONSTRAINTS = Authentication::Constraints::MultipleConstraint.new(
```

Comment:
IMO, all consts should be in one place.
Every authenticator has `consts.rb` file, follow  the pattern.

Reply:
Done